export/usb.py

```diff
@@ -3,13 +3,18 @@
 import re
 from subprocess import CalledProcessError, check_call
 import logging
-
 from pyudev import Context, Monitor
 
-video_pattern = re.compile('missed-moment.*merged.*mp4')
-MEDIA_DIR = '/missed_moment_media'
-USB_MOUNT_DIR = '/missed_moment_usb'
-USB_DIR = USB_MOUNT_DIR + '/missed_moment'
+# import constants
+import sys
+sys.path.append('..')
+import constants
```

Comment:
similar to the above, `sys` should be at the top (constants probably too).

Reply:
Sorry, `constants` is ours. It should be on it's own, but above `sys.path.append('..')`

Reply:
My confusion about `constants` reminds me we should just call it `config`. That way the use of env vars will read like `config.MEDIA_DIR`.

Reply:
I renamed constants.py to config.py and updated the necessary files for this change.